apps/services/endorsements/api/src/app/modules/endorsement/endorsement.controller.ts

```diff
@@ -27,6 +26,8 @@ import { IsEndorsementListOwnerValidationPipe } from '../endorsementList/pipes/i
 import { BulkEndorsementDto } from './dto/bulkEndorsement.dto'
 import { Endorsement } from './endorsement.model'
 import { EndorsementService } from './endorsement.service'
+import { Endorsement as EndorsementScope } from '@island.is/auth/scopes'
```

Comment:
See comment below about renaming the scopes enums to end with Scope.

Reply:
Will update 👍